proto/service.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package main;
+option go_package = "github.com/SandyWalsh/runner/gen";
+
+
+message LaunchRequest {
+    string cmd = 1;
+    repeated string arg = 2;
+}
+
+message Status {
+    enum StatusEnum {
+        ERROR = 0;
+        RUNNING = 1;
+        COMPLETED = 2;
+        ABORTED = 3;
+    }
+    StatusEnum status = 1;
+    int32 ExitCode = 2;
+}
+
+enum Error {
+    NOERROR = 0;
+    AUTHERROR = 1;   
+}
+
+message Output {
+    bytes data = 1;
+}
+
+message Process {
+    string pid = 1; 
```

Comment:
nit: I'd call the field `job_id` or something along those lines, so it doesn't get confused with the actual unix pid.

Reply:
fair ... naming is hard :)